nixos/modules/misc/ids.nix

```diff
@@ -356,6 +356,7 @@ in
       localtimed = 325;
       automatic-timezoned = 326;
       whisparr = 328;
+      chirpstack = 329;
```

Comment:
Can this not run with DynamicUser? We try to avoid allocating any new static ids, as explained in the comment at the top of this file.